symbols/overlay29.yml

```
@@ -8536,6 +8536,63 @@ overlay29:
         Called when the in-dungeon "others" menu is open. Does not return until the menu is closed.
         
         return: Always 0
+    - name: CheckVariousStatuses
+      address:
+        EU: 0x23016DC
+      description: |-
+        Returns 0 if none of these conditions holds for the given entity: asleep, frozen, petrified, biding.
+
+        r0: Entity pointer
+        return: bool
+    - name: CheckVariousStatuses2
+      address:
+        EU: 0x2301244
+      description: |-
+        Returns 0 if none of these conditions holds for the given entity: 
+        blinded (checked only if blind_check == 1), 
+        asleep, paused, infatuated, wrapping, wrapped, biding, petrified, or terrified.
+
+        r0: Entity pointer
+        r1: If true, return 1 if entity is blinded
+        return: bool
+    - name: IsChargingAnyTwoTurnMove
+      address:
+        EU: 0x2325084
+      description: |-
+        Returns a boolean indicating whether or not the given entity is charging any two-turn move.
+
+        r0: Entity pointer
+        r1: Unused boolean
+        return: bool
+    - name: CheckVariousConditions
+      address:
+        EU: 0x230156C
+      description: |-
+        Returns 0 if none of these conditions holds for the given entity: is a rescue client, 
+        doesn’t gain experience (a mission client/story teammate?), is a terrified non-team-leader, 
+        meets any of the conditions in CheckVariousStatuses2 (with blind_check = 0), is charging a two-turn move.
+
+        r0: Entity pointer
+        return: bool
+    - name: MonsterCanThrowItems
+      address:
+        EU: 0x2302D94
+      description: |-
+        Returns a boolean indicating whether or not the given monster can throw items based on its monster id.
+
+        r0: Monster pointer
+        return: bool
+    - name: GetPossibleAiItemTargets
+      address:
+        EU: 0x230FAA0
+      description: |-
+        Gets the positions of all targets that an AI can hit with a thrown item such as a Gravelerock.
+        The number of positions in the array will be stored at 0x023542FC.
+
+        r0: Entity pointer
+        r1: Item struct pointer
+        r2: Array of size 20 for storing position structs
```

Let's follow the convention for marking output variables:
```suggestion
        r2: [output] Array of size 20 for storing position structs
```